cirq/google/op_deserializer.py

```diff
@@ -136,3 +136,9 @@ def _args_from_proto(self, proto: v2.program_pb2.Operation
             if value is not None:
                 return_args[arg.constructor_arg_name] = value
         return return_args
+
+    def __eq__(self, other):
```

Comment:
Add tests of this behavior using `cirq.testing.EqualsTester`

Reply:
Removed.